src/main/java/org/javarosa/xpath/expr/XPathFormatDateForCalendarFunc.java

```diff
@@ -22,29 +23,41 @@ public XPathFormatDateForCalendarFunc(XPathExpression[] args) throws XPathSyntax
     }
 
     @Override
+    protected void validateArgCount() throws XPathSyntaxException {
+        if (args.length < 2 || args.length > 3) {
+            throw new XPathArityException(name, "2 or 3 arguments", args.length);
+        }
+    }
+
+    @Override
     public Object evalBody(DataInstance model, EvaluationContext evalContext, Object[] evaluatedArgs) {
-        return formatDateForCalendar(evaluatedArgs[0], evaluatedArgs[1]);
+        String formatString = null;
```

Comment:
Would it make sense to set this to `"%e %B %Y"` here (since it'll end up being that anyways if its not overwritten on the next line) or is that too cute?